### PR TITLE
Update metadata R packaging with new workshop structure

### PR DESCRIPTION
**Pull request by bvreede**
This PR revamps the text for the R packaging workshop, and updates the schedule and requirements to fit the new format (4 separate days with time in between to work on the growing package).

I am not sure whether the information about the time investment between sessions is in the right place (currently in Setup). Please advise :)

### CODE REVIEW

File: ds-rpackaging/schedule.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+This workshop is scheduled on four separate days. This allows you to work on
+your package in between, implementing the lessons you learned during our
```
Comment (mkuzak) [inline ds-rpackaging/schedule.md:2]:
```suggestion
your package in between the session, and implementing the lessons you learned during our
```

File: ds-rpackaging/schedule.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+meetings together. Please schedule time in your calendar to do so, so you can
```
Comment (mkuzak) [inline ds-rpackaging/schedule.md:3]:
```suggestion
meetings together. Please schedule dedicated work time in your calendar, so you can
```

**Comment by bvreede** [comment]
Suggestions implemented, thanks! Go ahead with the merge @mkuzak :)

File: ds-rpackaging/synopsis.md [hunk 3]
```diff
@@ -1 +1 @@
-In this workshop participants will take an R script, and make it into their own reusable, shareable R package.
\ No newline at end of file
+In this workshop participants will take an R script, and turn it into their own reusable, shareable R package.
```
Comment (mkuzak) [inline ds-rpackaging/synopsis.md]: `turn` is better. What about

> In this workshop participants will take their own R script and turn it into a reusable, shareable R package.

**Pull request merged**